definitions/divorce/json/State/State-deemed-and-dispensed-nonprod.json

```diff
@@ -30,5 +30,21 @@
     "Name": "Awaiting DN Application",
     "Description": "Awaiting DN Application",
     "DisplayOrder": 35
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "AwaitingServicePayment",
+    "Name": "Awaiting Service Payment",
+    "Description": "Awaiting Service Payment",
+    "DisplayOrder": 36
+  },
+  {
+    "LiveFrom": "04/08/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "ServicePayment",
+    "Name": "Service Payment",
```

Comment:
I think this state should be named better. It sounds like an event to me.

Reply:
Please make sure it's actually needed. It's only defined and permissions are granted.

I don't see it being used in any event.

Reply:
This was not renamed (nor was the comment responded to)